localstack/services/cloudformation/service_models.py

```diff
@@ -1759,11 +1759,13 @@ def _create_params(params, *args, **kwargs):
             tags = params.get("Tags") or []
             topic_name = params.get("TopicName")
             if dedup is not None:
-                attributes["ContentBasedDeduplication"] = dedup
+                attributes["ContentBasedDeduplication"] = (
+                    "true" if str(dedup).lower() == "true" else "false"
```

Comment:
nit: Looks like this could be a useful utility that could be used in different places. We could extract it into a small function in `common.py` (e.g. something like `canonicalize_bool_str(...)`).